modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -155,8 +158,8 @@ object NewPullRequestData {
 
   def ignoreFutureUpdates(update: Update): Details =
     Details(
-      "Ignore future updates",
-      s"""|Add this to your `${RepoConfigAlg.repoConfigBasename}` file to ignore future updates of this dependency:
+      "Suppress future updates",
```

Comment:
"Suppress" also make sense from user perspective, but I think "Ignore" should be kept as with the word in code base.

Reply:
Well it makes sense that it depends on who's perspective. _I_ suppress _Scala Steward_ by telling _it_ to ignore _me_

Reply:
are you guys inclining for "Suppress" or "Ignore"?
I personally don't like ignore word as it's kinda neglectful behavour, imho "Suppress" is more relevant in this given context